capa/features/extractors/vmray/__init__.py

```diff
@@ -128,41 +147,42 @@ def _compute_sections(self):
             for elffile_section in self.sample_file_static_data.elf.sections:
                 self.sections[elffile_section.header.sh_addr] = elffile_section.header.sh_name
 
-    def _compute_process_ids(self):
+    def _compute_monitor_processes(self):
         for process in self.sv2.processes.values():
-            # we expect VMRay's monitor IDs to be unique, but OS PIDs may be reused
-            assert process.monitor_id not in self.os_pid_by_monitor_id.keys()
-            self.os_pid_by_monitor_id[process.monitor_id] = process.os_pid
+            # we expect monitor IDs to be unique
+            assert process.monitor_id not in self.monitor_processes.keys()
```

Comment:
don't need `.keys()` here and in similar lines below

Reply:
https://github.com/mandiant/capa/pull/2396/commits/3cc48f18744baed375e09bd82bfc67478e531280

Reply:
Good catch!